Cargo.toml

```diff
@@ -14,7 +14,7 @@ resolver = "2"
 repository = "https://github.com/lalrpop/lalrpop"
 authors = ["Niko Matsakis <niko@alum.mit.edu>"]
 license = "Apache-2.0 OR MIT"
-version = "0.20.1" # LALRPOP
+version = "0.20.2"                                # LALRPOP
```

Comment:
```suggestion
version = "0.20.2" # LALRPOP
```

Reply:
Or is there a reason for this spacing? It's not very important but I personally think any kind of visual alignement (beyond basic indentation) in code will eventually not not work out as expected...

Reply:
No, I think it was just autoformatting... I'm not sure why it did that

Reply:
It's a strange formatting rule :sweat_smile:

Reply:
Sorry for the noise, it was my EvenBetterToml extension on VsCode that wants to align comments by default.